tools/systemd-units/char-rathena.service

```
@@ -0,0 +1,18 @@
+[Unit]
+Description=Rathena Char Server
+After=network-online.target remote-fs.target nss-lookup.target
+Wants=network-online.target
+#You might wanna edit the next line if your units are running in different servers instead of all in a single server.
+Requires=map-rathena.service login-rathena.service
+
+[Service]
+Type=single
+PIDFile=/run/char-serv.pid
+User=rathena
+ExecStart=${replacewithpath}/char-server
```

```suggestion
ExecStart=~/rathena/char-server
```
Since we specify a username as well, we can also specify the path inside his home directory or not?

---

Okay that is a good reason. Then we should do the same with `User=${replacewithyourrathenausername}` or something like this.